tests/wasm/README.md

```diff
@@ -0,0 +1,23 @@
+Create a C file containing
```

Comment:
We had all the wasm generation files in the tket repo so far, do we want to keep it that way?

Not sure what the best solution is, I think it would be good to test the extracted function signatures in the tket ci runs as well.

Reply:
I think it makes sense to have the source files here, as it allows easy editing and regeneration. Ideally maybe we'd just have the source files here and have the CI generate the wasm, but I think it's OK to have both in source control.

To an extent the function signatures are implicitly tested, since the calls to `add_wasm_to_reg()` would fail if they were incorrect.

Reply:
Sorry, I don't understand what you mean?

Reply:
The code for the wasm parsing and signature checks of the functions is in the tket repo. I think it would be good to have a wasm file similar to the two added in the PR tested there as well. That would make it possible to catch issues earlier when we change parts of the code in tket.

Reply:
Ah, I understand. Yes, I agree.